test/python/sdpa/test_torch_provider.py

```
@@ -0,0 +1,221 @@
+    cu = torch.nn.functional.pad(lens_t.cumsum(0), (1, 0)).to(torch.int32)
+    T, mx = int(cu[-1]), int(lens_t.max())
+    q = torch.randn(T, Hq, D, dtype=torch.bfloat16, device="cuda", requires_grad=True)
+    if kv_packed:  # non-contiguous k/v views of one buffer (token stride 2*H*D)
+        kv = torch.randn(T, 2, Hkv, D, dtype=torch.bfloat16, device="cuda", requires_grad=True)
+        k, v = kv[:, 0], kv[:, 1]
+    else:
+        k = torch.randn(T, Hkv, D, dtype=torch.bfloat16, device="cuda", requires_grad=True)
+        v = torch.randn(T, Hkv, D, dtype=torch.bfloat16, device="cuda", requires_grad=True)
+
+    fwd0, bwd0 = provider.calls["fwd"], provider.calls["bwd"]
+    out, lse = varlen_attn(q, k, v, cu, cu, mx, mx, window_size=window, enable_gqa=enable_gqa, return_aux=AuxRequest(lse=True))
```

Ruff (0.16.1)

[warning] Unpacked variable `lse` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)